src/contrib/cluster/Akka.Cluster.Sharding/ShardRegion.cs

```diff
@@ -294,21 +308,18 @@ protected object RegistrationMessage
             }
         }
 
-        /// <summary>
-        /// TBD
-        /// </summary>
+        /// <inheritdoc cref="ActorBase.PreStart"/>
```

Comment:
Yeah, I'm hoping they'll pick it up. In the meantime this is less ugly than having all of those TBDs